sultan/app/shared/product-categories/product-categories.js

```diff
@@ -1,12 +1,34 @@
 "use client";
 import React from "react";
+import { productCategoriesData } from "@/app/shared/product-categories-data.js";
 import "./product-categories.css";
 
 const ProductCategories = () => {
   const categoryButtonClick = () => {
     window.location.href = window.location.href;
   };
 
+  function MakeProductCategoriesList() {
+    const productCategoriesList = productCategoriesData.map(productCategory =>
+      <li key={productCategory.id}>
+        <button
+          onClick={categoryButtonClick}
+          className="product-categories-button"
+          id={productCategory.id}>
+          <div className="product-categories-button-background">
+            <img
+              className={productCategory.image_class}
+              src={productCategory.image}
+              alt={productCategory.name}
+            />
+          </div>
+          <p>{productCategory.name}</p>
+        </button>
```

Comment:
```suggestion
        <button
          onClick={categoryButtonClick}
          className="product-categories-button"
          id={productCategory.id}
        >
            <img
              className={productCategory.image_class}
              src={productCategory.image}
              alt={productCategory.name}
            />
          {productCategory.name}
        </button>
```

Можно ли сделать так?

Reply:
Если убрать <div className="product-categories-button-background">, то у картинки категории не будет фона, который меняется при клике, как в дизайне. Пока не придумали, как реализовать иначе